enhancements/platform-validation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Kepler Platform Validation
+authors:
+  - Jie Ren (jie.ren@intel.com)
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2023-05-27
+last-updated: 2023-05-27
+tracking-links: # link to related GitHub issues
+  - TBD
+---
+
+# Kepler Platform Validation Guide
+
+## Summary
+
+A quick start guide to validate Kepler support on specific hardware platform.
+
+## Motivation
+
+Currently, Kepler's test cases focus on code logic verification, lack of actual platform validation.
+
+Kepler's [data](https://github.com/sustainable-computing-io/kepler/tree/main/data) files majorly contain static information, or lack of data sync mechanism.
+
+Test cases and test report are not available, which could be the evidence or check criteria to below concerns:
+
+1. Whether the specific CPU/GPU/FPGA/Accelerator model is well supported in Kepler?
```

Comment:
not only for CPU, but also for memory and IO usage as disk also in kepler's goal, right? @marceloamaral , @rootfs ?

Reply:
what kind of memory/IO you are talking about ? those are common metric, do they ? 
there is no different model of CPU arch , GPU vendor etc?

Reply:
Agree, this is out of this proposal's scope, we may discuss in #645 topic.